easybuild/easyconfigs/c/CDO/CDO-1.9.0-CrayGNU-17.08.eb

```diff
@@ -18,10 +18,8 @@ sources = [SOURCELOWER_TAR_GZ]
 checksums = [ '2d88561b3b4a880df0422a62e5027e40' ]
 
 dependencies = [
-    ('netCDF', '4.4.1.1'),
-    ('netCDF-Fortran', '4.4.4'),
+    ('cray-netcdf/4.4.1.1.3', EXTERNAL_MODULE),
     ('cray-hdf5/1.10.0.3', EXTERNAL_MODULE),
-    # do not use cray-netcdf/4.4.1 because it is compiled with cray-hdf5/1.10
```

Comment:
I can't remember who but there's probably a user using hdf5/1.8.
Do you want to wait for the ticket ?

Reply:
I don't quite understand what you mean. I only updated the netCDF dependendency in this PR, I did not modify the hdf5 dependency...